lib/Importer/ONNX.cpp

```diff
@@ -68,16 +68,24 @@ bool ONNXModelLoader::loadProtoFile(onnx::GraphProto &net,
   return true;
 }
 
-size_t getPad(ArgumentDictionaryTy &dict) {
+std::vector<size_t> getPads(ArgumentDictionaryTy &dict) {
```

Comment:
since there is no modification to dict, we need explicitly show that with const X& dict.

Reply:
I tried to add "const", but there is compiler error: error: no viable overloaded operator[] for type 'const ArgumentDictionaryTy' (aka 'const unordered_map<basic_string<char, char_traits<char>, allocator<char> >, const onnx::AttributeProto *>')
    auto padStr = loadStr(dict["auto_pad"]);

Reply:
yup, see below how to properly fix this.